test/stat_tracker_test.rb

```diff
@@ -57,7 +57,33 @@ def test_it_has_game_teams
   # GAME STATISTICS
 
   def test_highest_total_score
-    assert_equal 5, @stat_tracker.highest_total_score
+
+    #Here I avoid loading the csv data all together
```

Comment:
Now, I'm able to test my method in isolation without a bunch of unnecessary data loaded up, and with mock objects that reliably do the one thing I need them to do for me to ensure my method runs properly.